omod/src/test/java/org/openmrs/module/fhir2/providers/r3/ObservationFhirResourceProviderWebTest.java

```diff
@@ -38,6 +38,7 @@
 import java.util.Objects;
 
 import ca.uhn.fhir.model.api.Include;
+import ca.uhn.fhir.rest.api.SortSpec;
```

Comment:
these test changes were not fun

Reply:
also seems like a lot of the r3/r4 tests are duplicated, wondering if it might be worth pulling out the duplicates into a base `ObservationFhirResourceProviderWebTest` test class and only include r3 vs r4 specific testing in these classes?